src/main/resources/db/migration/V22__add_type_to_job.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TYPE type AS ENUM ('BATCH', 'SERVICE', 'STREAM');
+
+ALTER TABLE jobs
+ADD COLUMN if not exists type TYPE;
+
+ALTER TABLE jobs
+ALTER COLUMN type SET DEFAULT 'BATCH';
```

Comment:
we don't need to set this at the database level, the code should already be setting this.